src/components/List/SimpleList/SimpleList.story.jsx

```diff
@@ -338,4 +340,36 @@ storiesOf('Watson IoT Experimental/SimpleList', module)
       </div>
     )),
     { info: { text: `` } }
-  );
+  )
+  .add('basic - SimpleList with reorder', () => {
```

Comment:
Could this just be named `with reorder` ? It looks like some of the stories are using `list with` as a convention so either of those would be easier to identify